common/protob/messages-cardano.proto

```diff
@@ -222,9 +222,17 @@ message CardanoSignTx {
 
 /**
  * Response: Serialised signed cardano transaction
- * @end
+ * @next CardanoSignedTxAck
```

Comment:
I'm not sure what the real purpose of these "annotations" is, but I feel like the `@end` should be kept here since this is the last message to be sent. I'm also basing this on `TxRequest` in `messages-bitcoin.proto`.

Reply:
makes sense - indeed I think at this point is close to what Ethereum implementation does but the respective messages are annotated in the same way

Reply:
done